net/wireguard-kmod/distinfo

```diff
@@ -1,3 +1,2 @@
-TIMESTAMP = 1620052580
-SHA256 (wireguard-freebsd-0.0.20210502.tar.xz) = cd59a0a711308083e40ac5e7e26abc63c8b1f4615a9d66f73f2398388b8e7b9b
-SIZE (wireguard-freebsd-0.0.20210502.tar.xz) = 50464
+SHA256 (wireguard-freebsd-0.0.20210503.tar.xz) = 96b2bceadc05dac2f026e203d1c0d25f114491afc8fbfc8d6f71c3de7f4a22f1
+SIZE (wireguard-freebsd-0.0.20210503.tar.xz) = 50548
```

Comment:
I already merged origin/main from upstream, please revert any change to this file

Reply:
Done